scripts/globals/crafting.lua

```diff
@@ -119,7 +321,7 @@ end
 -- tradeTestItem Action
 -----------------------------------
 xi.crafting.tradeTestItem = function(player, npc, trade, craftID)
-    local guildID    = craftID - 48
+    local guildId    = craftID - 48
```

Comment:
I know we normally fullLowerCamel, but I think an exception can be made for `ID` given its already short for "**Id**entification"

I really hate seeing `Id` anywhere. Just triggered the heck outa my OCD.

Reply:
We were using both `ID` and `Id` in this file. I have no real preference.

Reply:
I have a new, separate PR half ready touching this file (Guild Masters in this case)

If you guys want `ID` I promise tochange it in that one.